test/fixtures/scale.linear/min-max-skip/edge-case-3.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
   options: {
     spriteText: true,
     canvas: {
-      height: 213,
-      width: 536
+      height: 199,
+      width: 556
```

Comment:
Text is now cut off in these images. Is that expected?

Reply:
No, missed that. Will update

Reply:
Turns out the fixture comparison forces the new image to be same size as old image. So just needed to delete and re-create those.